src/docs/product/alerts/create-alerts/issue-alert-config.mdx

```diff
@@ -110,6 +114,10 @@ The available intervals are:
 - Hours: 3, 12, 24
 - Days: 7, 30
 
+## Alert Preview
+
+Displays what issues would have triggered a rule with your current conditions and when they would have last triggered it. Previews cannot be generated for certain rule sets and previews containing issue frequency conditions will be approximated.
```

Comment:
We should explain, at least at a high level, which alert rules won't generate preview.
```suggestion
The preview displays a list of issues that would have triggered the alert you've configured and when they would have last triggered it. Previews can't be generated for certain alert rules and previews containing issue frequency conditions are approximated.
```